packages/log-error/lib/index.js

```diff
@@ -47,7 +68,24 @@ function logError({ error, event, includeHeaders, level = 'error', request }) {
 		logData.request = serializeRequest(request, { includeHeaders });
 	}
 
-	logger.log(level, logData);
+	try {
+		logger[level](logData);
+	} catch (/** @type {any} */ loggingError) {
+		// We allow use of `console.log` here to ensure that critical
+		// logging failures are caught and logged. This ensures that we
+		// know if an app has broken logging.
+		// eslint-disable-next-line no-console
+		console.log(
+			JSON.stringify({
+				level: 'error',
+				event: 'LOG_METHOD_FAILURE',
+				message: `Failed to log at level '${level}'`,
+				error: serializeError(loggingError)
+			})
+		);
+		// eslint-disable-next-line no-console
+		console.log(JSON.stringify(logData));
```

Comment:
It will make it to Splunk but only if the app has Heroku log drains configured, and the only apps which have started to heavily adopt Reliability Kit are already migrated